### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by jaseyd** [letournel/path-finder#4]
Hi there, 

When I require "Letournel/PathFinder" : "~1.0" in my composer.json file and run composer install/update, I get the following error:

"letournel/pathfinder could not be found in any version, there may be a typo in the package name."

I'm able to install other packages successfully, am I doing something wrong here?

Cheers

**Comment by letournel** [comment]
Hi,

Thanks for using this small package. I hope you find it useful for your purposes.

I just tested on my machine, by typing :
````
composer require letournel/path-finder
````
And then it installs and downloads the package : 
````
Using version ^1.1 for letournel/path-finder
./composer.json has been created
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
  - Installing letournel/path-finder (1.1.0): Downloading (100%)
Writing lock file
Generating autoload files
````

Try to use lower cases, but I think it is simply due to the __missing dash between path and finder__.

Cheers :) !

**Comment by jaseyd** [comment]
Awesome. Installed now. Thanks for replying so soon!

**Comment by jaseyd** [comment]
I was following the instructions in the readme file under installation:

```
{
    "require": {
        "Letournel/PathFinder" : "~1.0"
    }
}
```

Does it need a hyphen adding here or am I missing something?

**Comment by letournel** [comment]
My bad, you're right there is a typo in the readme. It does need a hyphen. I just fixed that in PR #5 thank to your feedback !

**Comment by jaseyd** [comment]
No worries. Thanks for sorting that! :)
